src/modules/feed/apiClient.js

```diff
@@ -0,0 +1,68 @@
+import Settings from "src/config/settings";
+import { create } from "apisauce";
+
+const api = create({
+  baseURL: Settings.apiLocation,
+  headers: { Accept: "application/vnd.api+json" }
+});
+
+export const fetchTransactions = apiToken => {
+  const headers = {
+    "Content-Type": "application/x-www-form-urlencoded",
+    "Api-Token": apiToken
+  };
+
+  return new Promise(resolve => {
+    const request = api.get(
+      "transactions?include=activity,sender,receiver,votes&sort=-created_at",
+      {},
+      {
+        headers
+      }
+    );
+
+    request.then(response => {
+      var JSONAPIDeserializer = require("jsonapi-serializer").Deserializer;
```

Comment:
please use let or const. couldn't you just 

```javascript
import { Deserializer } from "jsonapi-serializer";
```

Reply:
Came across this one last monday; Fixed it in the next pull-request. Will fix this one though.